helm/deploy_test.go

```diff
@@ -53,25 +55,28 @@ func TestValidateRequiredResourcesForDeploy(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-// This is an end to end integration for the commands to setup helm access. This integrationtest is designed this way
+// This is an end to end integration for the commands to setup helm access. This integration test is designed this way
 // due to the way each step is setup to build on the previous step. For example, it is impossible to test grant without
 // having a helm server deployed, and configure without running grant.
 //
-// Test that we can:
+// Test that we can include:
```

Comment:
I think it was grammatically correct in the original form? Otherwise, I think it should say something like "Tests include:"

Reply:
My mistake, I misread it originally, will revert.